client/components/Dots.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons'
+library.add(faEllipsisH);
+import BoxButton from './BoxButton'
+
+class Dots extends React.Component {
```

Comment:
you're using `export default class`  in the other components it would be best to conform to one way of doing it and having the same for all the other classes

Reply:
Done.